README.md

```diff
@@ -30,6 +30,16 @@ requests need to be manually reviewed and merged by a human.
 It is ***highly recommended*** to also use [TagBot][tagbot], which automatically tags a release in your
 repository after the new release of your package is merged into the registry.
 
+### Requirements for packages in the General registry
+
+Packages registered in the General registry must be made in good faith.
```

Comment:
"Packages registered..." should probably be "Pull requests for packages to be registered..." - since the criterion relates to the PR, not the package itself (unless that is what you're trying to convey, in which case we might want to clarify.)

Is it the pull request that should be made in good faith, the package that must be made in good faith, or both?